src/QMCWaveFunctions/tests/test_pyscf_complex_MO.cpp

```diff
@@ -0,0 +1,905 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2018 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//
+// File created by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+/*  Author: Chandler Bennett
```

Comment:
This attribution can be moved to the "File created by" and/or "File developed by" header above.